grawlix-cms/_admin/book.page-create.php

```diff
@@ -287,10 +287,76 @@
 
 
 
+// Script for adding some HTML tags into the post
 $blog_output = <<<EOL
 <label for="blog_headline">Headline</label>
 <input type="text" name="blog_headline" id="blog_headline" value="$blog_headline"/>
 <label for="blog_post">Post</label>
+
+<script language="javascript" type="text/javascript">
+
+function addText(Text,Message)
+{
+  var obj = document.form.blog_post;
+
+  obj.focus();
+
+  if (document.selection && document.selection.createRange)  // Internet Explorer
+  {
+sel = document.selection.createRange();
+if (sel.parentElement() == obj)  sel.text = Text;
+  }
+
+  else if (typeof(obj) != "undefined")  // Firefox
+  {
+var longueur = parseInt(obj.value.length);
```

Comment:
Let's try to keep variable names in English, and descriptive. The variable name should tell us what this is a length *of*. If it's of the selection, perhaps call it `selLength`.